doc/sphinx/conf.py

```diff
@@ -0,0 +1,400 @@
+
+###########################
+# Set things up for XeTeX #
+###########################
+latex_elements = {
+    'babel': '',
+    'fontenc': '',
+    'inputenc': '',
+    'utf8extra': '',
+    'cmappkg': '',
+    # https://www.topbug.net/blog/2015/12/10/a-collection-of-issues-about-the-latex-output-in-sphinx-and-the-solutions/
+    'papersize': 'letterpaper',
+    'classoptions': ',openany', # No blank pages
+    'polyglossia' : '\\usepackage{polyglossia}',
+    'unicode-math' : '\\usepackage{unicode-math}',
+    'microtype' : '\\usepackage{microtype}',
+    "preamble": r"\usepackage{coqnotations}"
+}
+
+from sphinx.builders.latex import LaTeXBuilder
+
+########
+# done #
+########
+
+latex_additional_files = ["_static/coqnotations.sty"]
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+latex_documents = [
+    (master_doc, 'Coq85.tex', 'Coq 8.5 Documentation',
+     'The Coq Development Team (edited by C. Pit-Claudel)', 'manual'),
+]
+
+# The name of an image file (relative to this directory) to place at the top of
+# the title page.
+#latex_logo = None
+
+# For "manual" documents, if this is true, then toplevel headings are parts,
+# not chapters.
+#latex_use_parts = False
+
+# If true, show page references after internal links.
+#latex_show_pagerefs = False
+
+# If true, show URL addresses after external links.
+#latex_show_urls = False
+
+# Documents to append as an appendix to all manuals.
+#latex_appendices = []
+
+# If false, no module index is generated.
+#latex_domain_indices = True
+
+
+# -- Options for manual page output ---------------------------------------
+
+# One entry per manual page. List of tuples
+# (source start file, name, description, authors, manual section).
+man_pages = [
+    (master_doc, 'coq85', 'Coq 8.5 Documentation',
+     [author], 1)
+]
+
+# If true, show URL addresses after external links.
+#man_show_urls = False
+
+
+# -- Options for Texinfo output -------------------------------------------
+
+# Grouping the document tree into Texinfo files. List of tuples
+# (source start file, target name, title, author,
+#  dir menu entry, description, category)
+texinfo_documents = [
+    (master_doc, 'Coq85', 'Coq 8.5 Documentation',
+     author, 'Coq85', 'One line description of project.',
+     'Miscellaneous'),
+]
+
+# Documents to append as an appendix to all manuals.
+#texinfo_appendices = []
+
+# If false, no module index is generated.
+#texinfo_domain_indices = True
+
+# How to display URL addresses: 'footnote', 'no', or 'inline'.
+#texinfo_show_urls = 'footnote'
+
+# If true, do not generate a @detailmenu in the "Top" node's menu.
+#texinfo_no_detailmenu = False
+
+
+# -- Options for Epub output ----------------------------------------------
+
+# Bibliographic Dublin Core info.
+epub_title = project
+epub_author = author
+epub_publisher = author
+epub_copyright = copyright
```

Comment:
If this is useful this should be fixed; if not, probably cleaned up.

Reply:
I think an epub version of the manual isn't the top priority, but it might be nice? :)